lmfit/ui/ipy_fitter.py

```diff
@@ -116,12 +107,11 @@ def close(self):
         self.max_checkbox.close()
 
     def _repr_html_(self):
-        box = Box()
+        box = HBox()
         box.children = [self.value_text, self.vary_checkbox,
                         self.min_text, self.min_checkbox,
                         self.max_text, self.max_checkbox]
         display(box)
```

Comment:
By putting `box.add_class('hbox')` inside an `if` block like this we could keep compatibility with IPython 2.x.

```
if IPython.release.version_info[0] == 2:
    box.add_class('hbox')
```